src/os/bluestore/fastbmap_allocator_impl.h

```diff
@@ -578,6 +578,22 @@ class AllocatorLevel02 : public AllocatorLevel
     _mark_l2_on_l1(l2_pos, l2_pos_end);
     return allocated;
   }
+
+  void foreach_internal(
+    std::function<void(uint64_t offset, uint64_t length)> notify)
+  {
+    size_t alloc_size = get_min_alloc_size();
+    auto multiply_by_alloc_size = [alloc_size, notify](size_t off, size_t len) {
+      notify(off * alloc_size, len * alloc_size);
+    };
+    std::lock_guard l(lock);
+    l1.foreach_internal(multiply_by_alloc_size);
+  }
+  double get_fragmentation_internal() {
```

Comment:
@ifed01 Do you wish to start a new standard, replace prefix "_" with suffix "_internal" ?
Do you want me to chime in to this ?

Reply:
> @ifed01 Do you wish to start a new standard, replace prefix "_" with suffix "_internal" ? Do you want me to chime in to this ?

just missed that aspect. renaming to the original format...